README.md

```diff
@@ -41,6 +41,8 @@ Or run this command to deploy to a non-RBAC cluster:
 kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/master/deploy/infra/deployment.yaml
 ```
 
+> Important: if you are behind a firewall (e.g. Azure Firewall) and the Kubernetes Api Server is outside of your [AKS] cluster subnet, it is recommended to install AAD Pod Identity in the `kuebe-system` namespace by using the [helm charts]. Otherwise, please ensure the Kuberentes Api Server is explicity allowed adding a layer 4 rule to your Firewall.
```

Comment:
**Important**: For AKS clusters with limited [egress-traffic](https://docs.microsoft.com/en-us/azure/aks/limit-egress-traffic), Please install pod-identity in `kube-system` namespace using the [helm charts](https://github.com/Azure/aad-pod-identity/tree/master/charts/aad-pod-identity). 

@ferantivero wdyt?

Reply:
done